### PR TITLE
Add rules for schwarz-fritz.de

### PR DESCRIPTION
**Pull request by fabianschwarzfritz**
Fixes #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring existing code
- [x] New Ruleset
- [ ] Existing Ruleset

## For Rulesets: Testing

- [ ] Travis CI completed without errors (triggered upon pull request)

## Existing Contributors
If you have open PRs that have not been closed or merged, please revisit them by searching: `https://github.com/EFForg/https-everywhere/issues?q=is%3Aopen+assignee%3A[yourusernamehere]`.

### CODE REVIEW

**Comment by fabianschwarzfritz** [comment]
I see the jenkins build requires a `<test>` section. I will add one!

**Comment by zoracon** [comment]
@fabianschwarzfritz Here is a test example for your reference
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls

**Comment by pipboy96** [comment]
@fabianschwarzfritz Your website no longer has a valid certificate (neither on www.schwarz-fritz.de nor on schwarz-fritz.de). Also, it's recommended for the website owners to use [HSTS preloading](https://hstspreload.org/) instead of adding rulesets for their websites to HTTPS Everywhere since the preloaded websites get redirected to HTTPS without extensions on both Chrome and Firefox.

File: src/chrome/content/rules/schwarz-fritz.de.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="schwarz-fritz.de">
+  <target host="*.schwarz-fritz.de" />
+  <test url="http://www.schwarz-fritz.de/" />
+  <rule from="^http:" to="https:" />
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/schwarz-fritz.de.xml:5]:
```suggestion
<ruleset name="schwarz-fritz.de">
	<target host="schwarz-fritz.de" />
	<target host="www.schwarz-fritz.de" />

	<rule from="^http:" to="https:" />
</ruleset>
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]
Please fix the certificate issue and do not use wildcard targets. Otherwise it's fine, although it's recommended that you use HSTS preloading instead.

**Pull request closed without merging**